packages/esm-ward-app/src/config-schema.ts

```diff
@@ -133,4 +152,20 @@ export interface PatientAddressElementConfig {
   addressFields: Array<keyof PersonAddress>;
 }
 
-export type PatientCardElementConfig = {} & PatientAddressElementConfig;
+export interface PatientIdentifierElementConfig {
+  /**
+   * By default the preferred patient identifier is chosen,but
+   * if uuid is given the identifier corresponding to uuid is displayed
+   */
+  identifierTypeUuid: string | null;
+  /**
+   * Optional. The custom label or i18n key to the translated label to display for patient identifier. If not provided, defaults to the patient-identifier name.
+   * (Note that this can be set to an empty string to not show a label)
+   */
+  label?: string;
+  /**
+   * Optional. The custom module to use for translation of the label
+   */
+  labelI18nModule?: string;
+}
+export type PatientCardElementConfig = PatientIdentifierElementConfig & PatientAddressElementConfig;
```

Comment:
FYI this is a bit out of date (you will get a merge conflict). I made it so each element-specific config is a field in `PatientCardElementConfig`.

Reply:
Resolved the conflicts.Thanks @chibongho .